ethereumj-core/src/main/java/org/ethereum/crypto/ECKey.java

```diff
@@ -445,6 +540,31 @@ public static boolean validateComponents(BigInteger r, BigInteger s, byte v) {
             return true;
         }
 
+        public static ECDSASignature decodeFromDER(byte[] bytes) {
```

Comment:
The header javadoc for this class already says
```java
 * This code is borrowed from the bitcoinj project and altered to fit Ethereum.<br>
 * See <a href="https://github.com/bitcoinj/bitcoinj/blob/master/core/src/main/java/com/google/bitcoin/core/ECKey.java">
 * bitcoinj on GitHub</a>.
```

but I just realized that I wasn’t explicit here…
this decoding for a DER encoded ECDSA signature was also copied verbatim from BitcoinJ’s `ECKey`